bluephos/bluephos_pipeline.py

```diff
@@ -58,8 +59,11 @@ def rerun_candidate_generator(input_dir, t_nn, t_ste):
         DataFrame: A single-row DataFrame containing candidate data.
     """
     for file in Path(input_dir).glob("*.parquet"):
+        print(file.name)
```

Comment:
if you want to keep this, should probably make it a logger message

Reply:
That's the temporary code and should be removed. Thanks!